CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 Release notes
 =============
 
+Next Release
+------------------
+- Dark Mode toggle in the documentation.
```

Comment:
Please follow the convention used in other logs like this `We added dark mode toggle in the documentation`